pkg/universal_sigver/client.go

```diff
@@ -47,24 +47,24 @@ type Client interface {
 type backend struct {
 	provider          *ethclient.Client
 	smartWalletConfig *smart_wallet.Config
-	entryPointAddress *common.Address
+	entryPointAddress common.Address
 }
 
-func NewUniversalSigVer(providerURL string, smartWalletConfig *smart_wallet.Config, entryPointAddress *common.Address) Client {
+func NewUniversalSigVer(providerURL string, smartWalletConfig *smart_wallet.Config, entryPointAddress common.Address) (Client, error) {
```

Comment:
What's the reason of using config as a pointer?

Reply:
So that it is not copied to memory.

Reply:
It is a good practice to not use pointers, where they are not supposed to be

Reply:
Understood, thanks

Reply:
[1a7f206](https://github.com/layer-3/clearsync/pull/270/commits/1a7f20623aea652ebaf1504d4c9f7b53da5bb84e)